src/workloads/execution/BackgroundTTLDeletions.yml

```diff
@@ -132,7 +126,7 @@ Actors:
         Database: *db
         Threads: 1
         CollectionCount: *NumColls
-        DocumentCount: 50000
+        DocumentCount: 200000  # Add an initial set of documents. This will reduce noise in the next phases
```

Comment:
Consider using 200_000 for readability. It should just work, although I am also failing to find an example right now. Scientific notation works also 2e5.

Reply:
Didn't know you could do that in YAML!